CMakeLists.txt

```diff
@@ -95,14 +95,30 @@ target_compile_options(
     -Wall -Wextra -Wno-unused-parameter -Wno-type-limits -Werror>
 )
 
+if(${TRITON_ENABLE_METRICS})
+  target_compile_definitions(
+    triton-local-cache
+    PRIVATE TRITON_ENABLE_METRICS=1
+  )
+endif() # TRITON_ENABLE_METRICS
+
+if(${TRITON_ENABLE_LOGGING})
+  target_compile_definitions(
+    triton-local-cache
+    PRIVATE TRITON_ENABLE_LOGGING=1
+  )
+endif() # TRITON_ENABLE_LOGGING
+
 target_link_libraries(
   triton-local-cache
   PRIVATE
     triton-core-serverapi   # from repo-core
     triton-core-cacheapi    # from repo-core
     triton-core-serverstub  # from repo-core
+    triton-common-logging   # from repo-common
 )
 
+# TODO: Windows support
```

Comment:
Does it mean that local cache doesn't work on Windows? Did we support cache on Windows before the changes?